src/libertem/api.py

```diff
@@ -225,9 +265,9 @@ def create_radial_fourier_analysis(self, dataset, cx: float = None, cy: float =
             ri: float = None, ro: float = None, n_bins: int = None, max_order: int = None,
             use_sparse: bool = None):
         """
-        Calculate the Fourier transform of rings around the center.
+        Create an Analysis that calculates the Fourier transform of rings around the center.
```

Comment:
Good catch on the language use, here and the other instances!

Reply:
😃